### ISSUE TITLE
(accessibility) Screen reader say 'Clickable' when the split gutter is focused

### ISSUE THREAD

**Issue by albenakertova** [angular-split/angular-split#296]
**Describe the bug**
We're working on VPAT accessibility of our product. The split gutter cannot be accessed via Keyboard in normal tab flow. If the user goes in Browse mode after going through all the split areas, the Screen reader goes to the split gutter and announces 'Clickable'.

Expected: 

The screen reader should announce 'Separator' or other label to determine the role of the gutter. 

**To Reproduce**

1. Install NVDA (for Windows) or Voice over (for MAC)
2. Open Chrome browser with an example of <as-split>
2. Enter browse mode: e.g. for Windows + NVDA (insert + space), (NVDA key + space)
3. Navigate with up/down arrow keys to move between split areas.
4. Observe that the split gutter is announced in the end of the sequence, although it is in fact between the areas on the screen.
5. Observe that the gutter is announced just as 'Clickable'.

**Comment by bBaxta** [comment]
Regarding "The split gutter cannot be accessed via Keyboard in normal tab flow", this appears to be an issue in 13.2.0 build as well.

I've setup a Stackblitz using the demo app and adding buttons into the split areas to show the tab order:
https://stackblitz.com/edit/angular-split-demo-cgwewh?file=src%2Fapp%2Fapp.component.html
